README.md

```diff
@@ -44,8 +44,7 @@ sudo apt-get install openocd
   * ARM GNU Embedded Toolchain: https://developer.arm.com/open-source/gnu-toolchain/gnu-rm/downloads (Check `install_gcc_arm_none_eabi.sh` for which version to download)
   * STM32CubeMX: https://www.st.com/en/development-tools/stm32cubemx.html
   * MinGW: https://sourceforge.net/projects/mingw-w64/files/Toolchains%20targetting%20Win32/Personal%20Builds/mingw-builds/installer/mingw-w64-install.exe/download (**select 32-bit verison/i686 architecture**)
-  * Java: https://www.java.com/download/ie_manual.jsp
-
+  * Java: https://www.oracle.com/ca-en/java/technologies/javase/jdk11-archive-downloads.html
```

Comment:
link here may change. can you add a small note specifying that we get java 11? @pedrorgleite